.pre-commit-config.yaml

```diff
@@ -25,3 +25,26 @@ repos:
       files: "__init__.py"
       # ignore unused import error in __init__.py files
       args: ["--ignore=F401,E203", --config, setup.cfg]
+-   repo: https://github.com/pre-commit/mirrors-mypy
+    rev: v0.770
+    hooks:
+    - id: mypy
+      args: [
+          --follow-imports, silent
+        ]
+      files: |
+        (?x)^(
```

Comment:
I'm not too familiar with this syntax, but I have verified that `make typecheck` fails when I introduce the following fault:
```
diff --git a/workflows/prognostic_c48_run/runtime/loop.py b/workflows/prognostic_c48_run/runtime/loop.py
index 368d42ba..8222022b 100644
--- a/workflows/prognostic_c48_run/runtime/loop.py
+++ b/workflows/prognostic_c48_run/runtime/loop.py
@@ -50,6 +50,8 @@ from .names import AREA, DELP, TOTAL_PRECIP
 
 logger = logging.getLogger(__name__)
 
+a: int = "a"
+
 
 class Stepper(Protocol):
     """Stepper interface
```

Reply:
Yeah the syntax is confusing, basically it takes any regex. This regex in particular matches the regex patterns on each line below, which generally mean "all files under these directories".

That should cause a failure, shouldn't it? `workflows/prognostic_c48_run` was passed as an argument to mypy before.

Reply:
I was just verifying the code worked yah. More of an FYI.